docs/examples/use_cases/video_superres/dataloading/dataloaders.py

```diff
@@ -11,44 +11,47 @@
 
 from nvidia.dali.pipeline import Pipeline
 from nvidia.dali.plugin import pytorch
-import nvidia.dali.ops as ops
+import nvidia.dali.fn as fn
 import nvidia.dali.types as types
 
-class VideoReaderPipeline(Pipeline):
-    def __init__(self, batch_size, sequence_length, num_threads, device_id, files, crop_size):
-        super(VideoReaderPipeline, self).__init__(batch_size, num_threads, device_id, seed=12)
-        self.reader = ops.VideoReader(device="gpu", filenames=files, sequence_length=sequence_length,
-                                     normalized=False, random_shuffle=True, image_type=types.RGB,
-                                     dtype=types.UINT8, initial_fill=16, pad_last_batch=True)
-        self.crop = ops.Crop(device="gpu", crop=crop_size, dtype=types.FLOAT)
-        self.uniform = ops.Uniform(range=(0.0, 1.0))
-        self.transpose = ops.Transpose(device="gpu", perm=[3, 0, 1, 2])
-
-    def define_graph(self):
-        input = self.reader(name="Reader")
-        cropped = self.crop(input, crop_pos_x=self.uniform(), crop_pos_y=self.uniform())
-        output = self.transpose(cropped)
-        return output
+
+def get_video_reader_pipeline(batch_size, sequence_length, num_threads, device_id, files, crop_size):
```

Comment:
```suggestion
def create_video_reader_pipeline(batch_size, sequence_length, num_threads, device_id, files, crop_size):
```
or you can keep using a class, just use function API inside define_graph

Reply:
Done